x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/translations.tsx

```diff
@@ -29,17 +33,27 @@ export const NO_ACTIONS_READ_PERMISSIONS = i18n.translate(
   }
 );
 
-export const RULE_SNOOZE_DESCRIPTION = i18n.translate(
-  'xpack.securitySolution.detectionEngine.createRule.stepRuleActions.snoozeDescription',
+const ruleSnoozeDocsLink = `${
+  getDocLinks({ kibanaBranch: 'main' }).alerting.guide
+}#controlling-rules`;
+
+const RULE_SNOOZE_DOCS_LINK_TEXT = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepRuleActions.docsLinkText',
   {
-    defaultMessage:
-      'Select when automated actions should be performed if a rule evaluates as true.',
+    defaultMessage: 'docs',
   }
 );
 
-export const SNOOZED_ACTIONS_WARNING = i18n.translate(
-  'xpack.securitySolution.detectionEngine.createRule.stepRuleActions.snoozedActionsWarning',
-  {
-    defaultMessage: 'Actions will not be preformed until it is unsnoozed.',
-  }
+export const RULE_SNOOZE_DESCRIPTION = (
+  <FormattedMessage
+    id="xpack.securitySolution.detectionEngine.createRule.stepRuleActions.snoozeDescription"
+    defaultMessage="Select when automated actions should be performed. If a rule is snoozed actions will not be performed. Learn more about actions in our {docs}."
```

Comment:
Thanks for the additional context, @maximpn. I wasn't aware of that limitation but knowing it now, it makes sense why the new UI copy was added under the section title. Here's a few new options for you to choose from:

Option 1: Choose when to perform actions or snooze them. Notifications are not created for snoozed actions. Learn more

Option 2: Select when automated actions should be performed or snooze actions. Learn more 

I prefer the first one because I think it flows better and is more descriptive, but the second one works if you can't alter the first part of the sentence.

Reply:
I've updated the text with **Option 1**.